scripts/prepare-machine.ps1

```diff
@@ -44,6 +46,18 @@ if ($IsWindows) {
         reg.exe add $TlsClientKeyPath /v Enabled /t REG_DWORD /d 1 /f | Out-Null
     }
 
+    if ($EnableAppVerifier) {
+        # Enable Application Verifier for test binaries
+        where.exe appverif.exe
+        if ($LastExitCode -eq 0) {
+            appverif.exe /verify msquiccoretest.exe
+            appverif.exe /verify msquicplatformtest.exe
+            appverif.exe /verify msquictest.exe
+        } else {
+            Write-Debug "Application Verifier not installed!"
+        }
+    }
+
```

Comment:
Is it possible we could do this as a part of running the test, and not configuring the machine? The internal machines never even reboot (ideally), so these persist forever.

Reply:
It'll require the test script to run as administrator, and it'll blur the responsibilities of the scripts, but sure.  If the internal machines never reboot, why would having these settings always present be a problem?

Reply:
If you make it past of the test runner, then you can undo it when the test is done.